test/test_with_xpress.jl

```diff
@@ -1027,3 +1027,34 @@ end
 
 end
 
+@testset "PVspecs" begin
```

Comment:
since this new test does not require a solver I think that we should put it in the runtests.jl file, and that we should generally house tests that don't require a solver in runtests.jl (or a new file like unittests.jl that is `include`d in runtests.jl - in the future). Can you please move this test to runtests.jl and remove it from test_with_xpress & test_with_cplex ?

Reply:
addressed in https://github.com/NREL/REopt.jl/pull/71/commits/ed4eb816434cf961579a2b827c29525a5764c61c